### PR TITLE
Add Chennai, senthil's home town

### PR DESCRIPTION
**Pull request by senthil33**
This PR adds Chennai to the list of home towns.

@govind999 @santhimuthusamy @sswaminathanart

### CODE REVIEW

File: senthil-chennai.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#Chennai, TN
+
+###Restaurants
+Aasif biriyani
+Velu military 
+
+
```
Comment (sswaminathanart) [inline senthil-chennai.md:7]: All is well

**Review by sswaminathanart — APPROVED** [review]
look good

File: senthil-chennai.md (hunk 1 above)
Comment (govind999) [reply senthil-chennai.md:7]: why did you approve without talking to me

Comment (sswaminathanart) [reply senthil-chennai.md:7]: Please let me know when we can talk on this

**Review by sswaminathanart — COMMENTED** [review]
let me know the time to review it together

**Pull request merged**